owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt

```diff
@@ -1123,8 +1132,12 @@ class MainFileListFragment : Fragment(),
             }
 
             R.id.action_remove_file -> {
-                filesToRemove = checkedFiles
-                fileOperationsViewModel.showRemoveDialog(filesToRemove)
+                if (checkedFiles.size == 1 && !checkedFiles[0].isFolder) {
+                    context?.let { showRemoveCustomDialog(checkedFiles[0], it) }
```

Comment:
Instead of checking nullability of `context`, let's use better `requireContext()`, which is a function that always returns a non-nullable context or throws a `NullPointerException` (in this case there should always be a context, so NPE shouldn't be thrown ever):
```suggestion
                    showRemoveCustomDialog(checkedFiles[0], requireContext())
```